### PR TITLE
add nvidia gpu support

### PR DESCRIPTION
**Pull request by ZebRawnsley**
change base image from alpine to ubuntu, install ffmpeg5 from ppa and add required deps for nvidia

### CODE REVIEW

**Comment by ZebRawnsley** [comment]
Snippets:

ytdl_options:
```
      postprocessor_args:
        - "-c:v"
        - "h264_nvenc"
      merge_output_format: "mp4"
```

compose file:
```
    environment:
      - NVIDIA_DRIVER_CAPABILITIES=all
      - NVIDIA_VISIBLE_DEVICES=all
    deploy:
      resources:
        reservations:
          devices:
            - capabilities: [gpu]
```

**Comment by ZebRawnsley** [comment]
Fixes #388

File: docker/Dockerfile [hunk 1]
```diff
@@ -32,21 +44,22 @@ RUN mkdir -p /config && \
     # Install ytdl-sub, ensure it is installed properly
     pip install --no-cache-dir ytdl_sub-*.whl && \
       ytdl-sub -h && \
+      ldconfig && \
```
Comment (jmbannon) [inline docker/Dockerfile:47]: what does this do?

Comment (ZebRawnsley) [reply docker/Dockerfile:47]: while I was initially adding these changes I had some extra commands that didn't end up being needed, this line can be removed

**Comment by ZebRawnsley** [comment]
Another snippet

file_convert plugin:
  ```
convert:
    file_convert:
      convert_to: "mkv"
      convert_with: "ffmpeg"
      ffmpeg_post_process_args: >
        -c:v h264_nvenc
```

**Pull request closed without merging**